create_jenkins_job.py

```diff
@@ -186,13 +192,24 @@ def create_job(os_name, job_name, template_file, additional_dict):
 
         # configure nightly job for compiling with clang on linux
         if os_name == 'linux':
-            create_job(os_name, 'nightly_' + os_name + '_clang', 'ci_job.xml.em', {
+            create_job('linux', 'nightly_linux_clang', 'ci_job.xml.em', {
```

Comment:
There is no need to change this.

The next block should also use the variable `os_name` as part of the job name. That will allow enabling the same job on other platforms in the future without having to touch this again.

Reply:
Changed